executor/cte_test.go

```diff
@@ -19,64 +19,67 @@ import (
 	"fmt"
 	"math/rand"
 	"sort"
+	"testing"
 
-	"github.com/pingcap/check"
-
+	"github.com/pingcap/failpoint"
+	"github.com/pingcap/tidb/config"
 	"github.com/pingcap/tidb/domain"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/session"
 	"github.com/pingcap/tidb/sessionctx"
-
-	"github.com/pingcap/failpoint"
-	"github.com/pingcap/tidb/config"
 	"github.com/pingcap/tidb/store/mockstore"
+	"github.com/pingcap/tidb/testkit"
 	"github.com/pingcap/tidb/util/mock"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/stretchr/testify/require"
 )
 
-var _ = check.Suite(&CTETestSuite{&baseCTETestSuite{}})
-var _ = check.SerialSuites(&CTESerialTestSuite{&baseCTETestSuite{}})
-
-type baseCTETestSuite struct {
+type CTETestSuite struct {
 	store      kv.Storage
 	dom        *domain.Domain
 	sessionCtx sessionctx.Context
 	session    session.Session
 	ctx        context.Context
+	close      func()
 }
 
-type CTETestSuite struct {
-	*baseCTETestSuite
-}
-
-type CTESerialTestSuite struct {
-	*baseCTETestSuite
-}
-
-func (test *baseCTETestSuite) SetUpSuite(c *check.C) {
+func SetUpSuite(t *testing.T) *CTETestSuite {
```

Comment:
```suggestion
func setUpSuite(t *testing.T) *CTETestSuite {
```

We don't export it, do we?